src/main/groovy/io/seqera/wave/service/persistence/WaveContainerRecord.groovy

```diff
@@ -173,6 +179,7 @@ class WaveContainerRecord {
         this.buildId = data.buildId
         this.buildNew = data.buildId ? data.buildNew : null
         this.freeze = data.buildId ? data.freeze : null
+        this.fusionVersion = request?.containerConfig?.fusionVersion()
```

Comment:
Let's add a test in https://github.com/seqeralabs/wave/blob/ffa32c5ec97e3a379846a8e724e548b086bc8c6d/src/test/groovy/io/seqera/wave/service/persistence/WaveContainerRecordTest.groovy#L38-L38